zmq.cpp

```diff
@@ -16,13 +17,6 @@ std::string z_recv(zmq::socket_t& socket) {
     return std::string(static_cast<char*>(msg.data()), msg.size());
 }
 
-namespace {
-
-bool is_valid_message(const std::vector<zmq::message_t>& frames) {
```

Comment:
You removed this method because of the const?

Reply:
yep, it's a small funtion..

Reply:
and i hate non const...